src/altscore/borrower_central/__init__.py

```
@@ -1,5 +1,7 @@
 from altscore.borrower_central.model.borrower import BorrowersAsyncModule, BorrowersSyncModule
 from altscore.borrower_central.model.category import CategoryAsyncModule, CategorySyncModule
+from altscore.borrower_central.model.conversational_templates import WhatsAppTemplateSync, WhatsAppTemplateSyncModule, \
```

Ruff (0.12.2)

`altscore.borrower_central.model.conversational_templates.WhatsAppTemplateSync` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)